eng-paths/software_engineer_3.md

```diff
@@ -228,6 +228,7 @@ Sr Software Engineer 1
 - Communicates well both in writing and verbally with teammates, peers, and stakeholders.
 - Positively influences team culture.
 - During critical outages and high-impact issue investigations, make thoughtful and clear communication regarding how to achieve the proper outcomes.
+- Able to communicate trade-offs with PM and EM, and drive Engineering improvements within the team scope.
```

Comment:
This would be across the board, but WDYT about clarifying the responsibility of discovery and understanding those?
- Able to discover, analyze, understand, and communicate trade-offs with PM and EM, and drive Engineering improvements within the team scope.

Reply:
I like @diogormendes' suggestion here!

Reply:
liked it also, done! ✓ @diogormendes I just won't copy across the board because the idea is that this expectation will be carried for the levels "above" :)

Reply:
Sure Paula 😀